src/main/java/com/thevoxelbox/voxelsniper/brush/type/AbstractBrush.java

```diff
@@ -91,11 +93,24 @@ public int getHighestTerrainBlock(int x, int z, int minY, int maxY) {
         }
     }
 
-    public void regenerateChunk(int chunkX, int chunkZ) {
+    public boolean regenerateChunk(int chunkX, int chunkZ) {
         try {
             World world = BukkitAdapter.adapt(editSession.getWorld());
-            editSession.regenerateChunk(chunkX, chunkZ, editSession.getBiomeType(chunkX << 4, 0, chunkZ << 4),
-                    world.getSeed()
+            int minX = chunkX << 4;
+            int minZ = chunkZ << 4;
+            RegenOptions regenOptions = RegenOptions.builder()
+                    .seed(world.getSeed())
+                    .regenBiomes(true)
+                    .biomeType(null)
```

Comment:
I'm not familiar with brush functionality in FAVS, but would it be more appropriate for this to have a configurable biome type for this method rather than a static null? The biomeType was implemented in FAWE's RegenOptions to allow for specific areas to be regenerated using the settings of a specified biome (null simply regenerating the chunk as it was originally generated). Not necessary for approval, but something to think about!

Reply:
Oh, excellent idea, ty! It could be included in another pr, or added in this one 🤷

Reply:
If you want, you can do it here and I'll pull it later today.